src/components/configurable-columns/configurable-columns.scss

```diff
@@ -0,0 +1,16 @@
+.carbon-configurable-columns {
+  .carbon-form__buttons {
+    position: relative;
+  }
+
+  .carbon-form__additional-actions {
+    position: absolute;
+    left: 0;
```

Comment:
Not for this PR but we should take a think of how to do this via the component - seen this done a few times now.